modules/quality/src/qualitybrisque.cpp

```diff
@@ -53,6 +54,16 @@ namespace
     // brisque intermediate matrix element type.  float if BRISQUE_CALC_MAT_TYPE == CV_32F, double if BRISQUE_CALC_MAT_TYPE == CV_64F
     using brisque_calc_element_type = float;
 
+    // returns path to a file located in the opencv install/data path
+    cv::String get_install_data_path(const cv::String& file_name)
+    {
+#if !defined( OPENCV_INSTALL_PREFIX ) || !defined( OPENCV_DATA_INSTALL_PATH )
+        return cv::String();
+#else
+        return cv::String(OPENCV_INSTALL_PREFIX) + "/" + OPENCV_DATA_INSTALL_PATH + "/quality/" + file_name;
```

Comment:
It is broken due missing files in "guessed" location.

Installation prefix makes no sense on Windows (final location on users system is not fixed).

Lets avoid handling of "empty" model case (tests / samples should pass valid path).